### DISCUSSION TITLE
Semantic Kernel third party plugin support

### DISCUSSION THREAD

**Discussion by jsypkens** [microsoft/semantic-kernel#1348] [Q&A] [answered]
https://www.theverge.com/2023/5/23/23732505/microsoft-ai-bing-copilot-chatgpt-interoperability-support-build

At Build 2023, Microsoft announced that plugins would be interoperable between Bing Chat, ChatGPT, and Semantic Kernel. Does that mean that we can use plug-ins built for ChatGPT inside of semantic kernel?  If so, how? If not, then what does interoperable plugins exactly mean? Is there a plug-in repository where we could find and pull in plug-ins for semantic kernel, or are we only intended to use the out-of-the-box plugins and build our own?

**Comment by craigomatic** [comment, marked as answer]
Yes you can "import" the plugin to SK. A good place to start looking at the code to do this would be over here (C#): https://github.com/microsoft/semantic-kernel/blob/main/samples/dotnet/kernel-syntax-examples/Example21_ChatGPTPlugins.cs

**Reply by jsypkens** [reply]
Thanks Craig! Unsurprisingly, most of the plug-ins we're finding require API keys to work. Wondering if you (or anyone) have found any plug-ins that can actually be used with this approach.
